src/components/common/inputs/Input/Input.tsx

```diff
@@ -96,7 +97,9 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
             />
             {isFocused && value.length > 0 && (
               <div css={deleteButtonStyle} onMouseDown={handleInputDelete}>
-                <IcDelete20 />
+                <span css={deleteIconStyle}>
+                  <IcDelete20 />
+                </span>
```

Comment:
```suggestion
                <IcDelete20 css={deleteIconStyle}/>
```

이거 이런식으로 svg 태그에다가 css직접 넣어도 돼여